joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50
  
arm1_trajectory_controller:
    type: position_controllers/JointTrajectoryController
    joints:
        - arm1_a_joint
        - arm1_b_joint
    constraints:
          goal_time: 0.6
          stopped_velocity_tolerance: 0.05
          joint1: {trajectory: 0.1, goal: 0.1}
          joint2: {trajectory: 0.1, goal: 0.1}
    stop_trajectory_duration: 0.5
    state_publish_rate:  25
    action_monitor_rate: 10

arm2_trajectory_controller:
    type: position_controllers/JointTrajectoryController
    joints:
        - arm2_a_joint
        - arm2_b_joint
    constraints:
          goal_time: 0.6
          stopped_velocity_tolerance: 0.05
          joint1: {trajectory: 0.1, goal: 0.1}
          joint2: {trajectory: 0.1, goal: 0.1}
    stop_trajectory_duration: 0.5
    state_publish_rate:  25
    action_monitor_rate: 10